py3status/modules/i3pystatus.py

```diff
@@ -0,0 +1,328 @@
+# -*- coding: utf-8 -*-
+"""
+Support i3pystatus modules in py3status.
+
+[i3pystatus](https://github.com/enkore/i3pystatus) is an alternative to
+py3status and provides a variety of modules.  This py3status module allows
+these modules to run and be display inside py3status.
+
+The modules parameters are provide as such
+```
+i3pystatus clock {
+    module = 'clock'
+    format = [ ('%a %b %-d %b %X', 'America/New_York'), ('%X', 'Etc/GMT+9') ]
+}
+```
+
+if backend(s) are provided they should be given as a dict with the key being
+the backend name and the value being a dict of the backend settings
+
+```
+i3pystatus weather {
+    module = 'weather'
+    format = '{condition} {current_temp}{temp_unit}[ {icon}]'
+    format += '[ Hi: {high_temp}][ Lo: {low_temp}][ {update_error}]'
+    backend = {
+        'weathercom.Weathercom': {
+            'location_code': '94107:4:US',
+            'units': 'imperial',
+        }
+    }
+}
+```
+
+backends that have no configuration should be defined as shown here
+
+```
+i3pystatus updates{
+    module = 'updates'
+    backends = {'dnf.Dnf': {}}
+}
+```
+
+Configuration parameters:
+    module: name of the i3pystatus module (default None)
+
+    (additional parameters as per the i3pystatus module)
+
+Requires:
+    i3pystatus:
```

Comment:
(Suggestion) `i3py3status: i3status replacement written in python`

Reply:
done